stl/src/syncstream.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
As this separately compiled source file is being injected into the import lib, we should have the Scary Comment For Import Lib Stuff:
https://github.com/microsoft/STL/blob/47cd70315d560e4fe1ac699c0eb8ec8bfc5cb658/stl/src/vector_algorithms.cpp#L4-L7
(This is moot if we move this into a satellite DLL.)